doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md

```diff
@@ -11,63 +10,121 @@ sort_idx: 10
+* **Authentication credentials** -- credentials used to authenticate on Cassandra cluster.
+* **Keyspace name** -- Cassandra keyspace used to prefix the data table.
+* **Table name pattern** -- pattern used to create table name (e.g. `logs_$app_token` adds the application token at the end of the table name).
+* **Column mapping** -- mapping of specific log data to appropriate columns.
+* **Clustering** -- clustering key configuration.
 
-An example configuration that matches to previously introduced Avro schema is as below:
+|Type           |Example value                                                                  |Example column type|Description                                                         |
+|---------------|-------------------------------------------------------------------------------|-------------------|--------------------------------------------------------------------|
+|`HEADER_FIELD`   |endpointKeyHash, applicationToken, headerVersion,timestamp, logSchemaVersion   |TEXT               |Maps a header variable to the specified column.                   |
+|`EVENT_FIELD`    |telemetry                                                                      |DOUBLE             |Maps a log schema field to the specified column.                  |
+|`CLIENT_FIELD`   |clientField                                                                    |TEXT               |Maps a client-side endpoint profile field to the specified column.|
+|`SERVER_FIELD`   |serverField                                                                    |TEXT               |Maps a server-side endpoint profile field to the specified column.|
+|`${TYPE}_JSON`   |                                                                               |TEXT               |Maps one of the fields listed above as a corresponding JSON.      |
+|`${TYPE}_BINARY` |                                                                               |BLOB               |Maps one of the fields listed above in binary format.             |
+|`UUID`           |                                                                               |UUID               |Maps a UUID generated by Kaa to the specified field.              |
+|`TS`             |yyyy-MM-dd-'Time:'HH:mm:ss.SS                                                  |TEXT               |Maps a timestamp generated by Kaa to the specified field. Timestamp pattern is handled by java.text.SimpleDateFormat |
+
+Use the checkboxes to make any field a partition and/or clustering key.
+
+Configure the key clustering in the **Clustering** section by adding column names and setting their order (**DESC** or **ASC**).
+
+>**NOTE:** You can specify Cassandra driver settings in the log appender configuration.
+>See [official documentation](http://docs.datastax.com/en/landing_page/doc/landing_page/current.html).
+{:.note}
+
+Below is an example configuration that matches the mentioned Avro schema.
```

Comment:
Look at the configuration below. Some fields and their values contain a needless space at the end. Error occurs while uploading  such configuration in Administration UI. Please remove these spaces.